packages/generators/__tests__/init-generator.test.ts

```diff
@@ -12,6 +12,16 @@ describe('init generator', () => {
             stylingType: 'No',
             useExtractPlugin: 'main',
         });
+
+        // Check that all the project files are generated with the correct name
+        const filePaths = ['package.json', 'README.md', 'src/index2.js'];
+        assert.file([...filePaths.map(file => join(outputDir, file))]);
```

Comment:
I had to skip the other tests that I merged because they were failing. You should use the jest assertion in order to tell to jest that is working. Or checking `yeoman-test` documentation to understand how to integrate it with jest

Reply:
Okay, sounds fair, will take a look. 👍

Reply:
Is it a problem with yeoman-assert? I can't seem to find a way to integrate them with jest and the docs are really minimal.

Reply:
Triggered travis on next again and it seems to pass now. 😟🤔

Reply:
Check the logs locally while you test them (maybe run that single test). There are errors and jest can't recognize any assertion so it's like the test is empty

Reply:
At this point do not use `yeoman-assert` and stick with the normal `expect`

Reply:
> Check the logs locally while you test them (maybe run that single test). There are errors and jest can't recognize any assertion so it's like the test is empty

I just ran a test which was supposed to fail in yeoman-assert and jest caught it, I think the errors from yeoman-assert are caught by jest on exit.